### PR TITLE
Lambdafy Nest + Reorg package names + TF Cert Request 

### PR DESCRIPTION
**Pull request by vyasworks**

### CODE REVIEW

File: .example.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+RUNTIME_ENV=local
```
Comment (hitankar-freshworks) [inline .example.env:1]: its standard to have example file names as `.env.example`

File: apps/api/package.json [hunk 2]
```diff
@@ -23,17 +23,20 @@
...
   },
   "devDependencies": {
     "@nestjs/cli": "^8.0.0",
     "@nestjs/schematics": "^8.0.0",
     "@nestjs/testing": "^8.0.0",
+    "@types/aws-lambda": "^8.10.89",
     "@types/express": "^4.17.13",
     "@types/jest": "27.0.2",
-    "@types/node": "^16.0.0",
+    "@types/node": "^17.0.8",
```
Comment (hitankar-freshworks) [inline apps/api/package.json:39]: v17? We need to coordinate on node version to use.

**Review by hitankar-freshworks — APPROVED** [review]
Approving this, but please change `.example.env` to `.env.example`.

File: apps/api/package.json (hunk 2 above)
Comment (vyasworks) [reply apps/api/package.json:39]: not sure how it got updated. ill remove this

**Pull request merged**